.circleci/config.yml

```diff
@@ -1439,7 +1435,6 @@ workflows:
             requires:
               - javadoc-generation
             pre-steps:
-              - install-tools
```

Comment:
I believe we need extra installations on services side. Removing this will fail CI. (I could see the CI step failed with the changes.)